src/base/_foundation.lua

```diff
@@ -334,7 +334,10 @@
 		if _OPTIONS.fatal then
 			error(message)
 		else
+			local color = term.getTextColor();
+			term.setTextColor(13);
```

Comment:
Maybe instead of using the value explicitly, we could create a `term.warningColor` that can be set based on the OS?